azext_iot/_params.py

```diff
@@ -761,6 +761,87 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+    
+    with self.argument_context("iot hub state export") as context:
+        context.argument(
+            "hub",
+            options_list=["--hub-name", "-n"],
+            help="IoT Hub name."
+        )
+        context.argument(
+            "rg",
+            options_list=["--resource-group", "-rg"],
+            help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`."
+        )
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file where the hub and device information will be stored."
+        )
+        context.argument(
+            "auth_type_dataplane",
+            options_list=["--auth-type"],
+            arg_type=hub_auth_type_dataplane_param_type,
+        )
+
+    with self.argument_context("iot hub state import") as context:
+        context.argument(
+            "hub",
+            options_list=["--hub-name", "-n"],
+            help="IoT Hub name."
+        )
+        context.argument(
+            "rg",
+            options_list=["--resource-group", "-rg"],
+            help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`."
+        )
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file from which the hub and device information will be read."
+        )
+        context.argument(
+            "auth_type_dataplane",
+            options_list=["--auth-type"],
+            arg_type=hub_auth_type_dataplane_param_type,
+        )
+        context.argument(
+            "overwrite",
+            options_list=["--overwrite"],
+            help="If this flag is set, then the import will overwrite all configurations and devices in the destination hub."
+        )
+
+    with self.argument_context("iot hub state migrate") as context:
+        context.argument(
+            "dest_hub",
+            options_list=["--dest-hub", "-n"],
```

Comment:
for longer names, like destination hub, spell them out. Then the short hands usually are just the first letters of the words: --dh
for destination hub resource group -> --dhg (cause -g == resource group)